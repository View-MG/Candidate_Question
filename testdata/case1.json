[
  {
    "no": 1,
    "platformProductId": "FG0A-CLEAR-IPHONE16PROMAX",
    "qty": 2,
    "unitPrice": 50,
    "totalPrice": 100
  }
]