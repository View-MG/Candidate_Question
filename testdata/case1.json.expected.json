[
  {
    "no": 1,
    "productId": "FG0A-CLEAR-IPHONE16PROMAX",
    "materialId": "FG0A-CLEAR",
    "modelId": "IPHONE16PROMAX",
    "qty": 2,
    "unitPrice": 50.00,
    "totalPrice": 100.00
  },
  {
    "no": 2,
    "productId": "WIPING-CLOTH",
    "qty": 2,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 3,
    "productId": "CLEAR-CLEANNER",
    "qty": 2,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  }
]